CI/integration_tests/visualizer/znvis_visualizer.py

```diff
@@ -0,0 +1,94 @@
+"""
+MDSuite: A Zincwarecode package.
+
+License
+-------
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v2.0 which accompanies this distribution, and is
+available at https://www.eclipse.org/legal/epl-v20.html
+
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zincwarecode Project.
+
+Contact Information
+-------------------
+email: zincwarecode@gmail.com
+github: https://github.com/zincware
+web: https://zincwarecode.com/
+
+Citation
+--------
+If you use this module please cite us with:
+
+Summary
+-------
+Test that the visualizer runs.
+"""
+import os
+import tempfile
+import time
+import unittest
+
+from zinchub import DataHub
+
+import mdsuite as mds
+from mdsuite.utils.testing import MDSuiteProcess
+
+
+class TestZnvisVisualizer(unittest.TestCase):
+    """
+    A test class for the Particle class.
+    """
+
+    def test_run(self):
+        """
+        Run the simple spheres tutorial and ensure it does not throw
+        exceptions.
+
+        Returns
+        -------
+
+        Notes
+        -----
+        TODO: We can set up a means of havng the main thread check if the test is
+            finished on the deployed process and then kill it. That will ensure that the
+            test isn't prematurely closed and passes when it should have failed. For now
+            120 seconds is more than enough time for this test but we can think of some
+            improvements later.
+        """
+        process = MDSuiteProcess(target=self._run_app)
+        process.start()
+        time.sleep(90)  # give it 90 seconds to run.
+        process.terminate()
```

Comment:
Why 90 s?

Reply:
I do explain why there is a time here in the TODO. It is a different time in the todo but the concept is the same. 

The point here is that after enough time for the test to run the process must be killed. Right now I don't include communication from the process back to the main test and so I simply wait for 90 seconds, enough time for the test to run, and then kill it. In future, we can work on a way to communicate between the threads but as long as the process test is not long this is fine.